spot_wrapper/wrapper.py

```diff
@@ -1302,6 +1304,20 @@ def velocity_cmd(
         self.last_velocity_command_time = end_time
         return response[0], response[1]
 
+    def arm_joint_cmd(
+        self, sh0: float, sh1: float, el0: float, el1: float, wr0: float, wr1: float
```

Comment:
```suggestion
        self, *, sh0: float, sh1: float, el0: float, el1: float, wr0: float, wr1: float
```

When you have a bunch of arguments with the same type that can easily get swapped, I would recommend keyword guarding them.

Reply:
..... i'll be honest, not sure why we need the `*` input here...
also, I just tested it and it is keyword guarded isn't it?

Reply:
lol nevermind. Updated